### DISCUSSION TITLE
Mobaco goes virtual

### DISCUSSION THREAD

**Discussion by KoosWelling** [leozide/leocad#735] [Show and tell]
Hello,
Ever heard from Mobaco?
It's a vintage building construction toy, sold during 1924-1960 in multiple countries.
It consists of base plates, wooden pillars & cardboard panels and with a handy floorplan, you are able to create big buildings.

With a small Mobaco fan club, we like to transfer this Mobaco construction toy into the digital world.
And one of the best ways, seems to be LeoCAD:
Frist of all it's free to use (thank you) and it's multi platform (thank you) and it's uses a 'standard' LDraw format (which is also well supported) and also LeoCAD is still maintained (thank you).

Hereby a small impression, how the LeoCAD is 'used' for our needs.
![Moubal_Manual_Buildings](https://user-images.githubusercontent.com/66367852/130143657-9bc34068-900b-4932-a5f0-5fcfb1806f5f.jpg)
![Mobaco_Model_39](https://user-images.githubusercontent.com/66367852/130143663-a276eb6b-7df0-4816-ba67-09ca1d2b15cf.png)
![Mobaco_Model_47](https://user-images.githubusercontent.com/66367852/130143666-a961faba-dd05-42a3-8ca6-981152ee597b.png)
Click for more [information](https://mol8.home.xs4all.nl/MOBACO_Seamonkey/Home.html) about Mobaco.

**Comment by KoosWelling** [comment]
Included the complete STL & DAT library here, including the Phyton script converting STL's to LDraw (DAT):
https://github.com/KoosWelling/Mobaco

**Comment by leozide** [comment]
This is pretty cool, well done!

**Comment by MrEdgeNL** [comment]
After an interesting year, being busy with virtualizing Mobaco, the complete library is done.

The library became much more, including adding never produced parts and rebuild those in real life.
It's als interesting to see the timeline of the product.

And on top of that, also many other building systems are virtualized (will be released later).
All these systems could be used in one LeoCAD environment, which is pretty cool!


It's funny to realize, this wooden + cardboard toy, is not so different then Lego's. (Especially in digital environment.) 
They are both excelent ways of: imagination, creation and having fun. 

For more information about the library and how to add custom LDraw files, see also:
https://retrobuildingtoys.nl/category/vintage_toys/vintage-toys-collection/
